cosmic-applet-status-area/src/components/status_menu.rs

```diff
@@ -118,15 +121,48 @@ impl State {
     }
 
     pub fn opened(&self) {
-        let menu_proxy = self.item.menu_proxy().clone();
+        let Some(menu_proxy) = self.item.menu_proxy().cloned() else {
+            let item_proxy = self.item.item_proxy().clone();
+            tokio::spawn(async move {
+                if let Err(e) = item_proxy.activate(0, 0).await {
```

Comment:
Oh yeah, we need to figure out how to supply sensible coordinates here.

I believe it's the coordinates the item was clicked at, or at least based on that.